src/main/java/org/broadinstitute/consent/http/service/DaaService.java

```diff
@@ -102,6 +109,33 @@ public DataAccessAgreement findById(Integer daaId) {
     throw new NotFoundException("Could not find DAA with the provided ID: " + daaId);
   }
 
+  public void sendDaaRequestEmails(User user, Integer daaId) throws Exception {
+    try {
+      List<Institution> institutions = institutionService.findAllInstitutions();
+      Institution institution = institutions.stream()
+            .filter(inst -> inst.getId().equals(user.getInstitutionId()))
```

Comment:
Could `inst.getId()` ever be null?  If so, this would yield an NPE, similar to @rushtong 's recent PR.  Is there a reason not to compare with `==`?

Reply:
The id in this case is a database id, it should never be null here. If it is, there is a deeper problem we need to address